examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>How is that obvious? There are  hundreds of digits. And the <code>obvious</code>  result is literary buried among them.</p><p>Make the obvious obvious</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>